homeassistant/components/camera/synology.py

```diff
@@ -146,7 +151,7 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
             tasks.append(device.async_read_sid())
             devices.append(device)
 
-    yield from asyncio.wait(tasks, loop=hass.loop)
+    yield from asyncio.gather(*tasks, loop=hass.loop)

```

Comment:
No idea.  I don't think I changed this, I must have started with the wrong code base.